LipstickTests/UIColor+ModificatorsTest.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  UIColor+ModificatorsTest.swift
+//  Lipstick
+//
+//  Created by Filip Dolnik on 18.10.16.
+//  Copyright © 2016 Brightify. All rights reserved.
+//
+
+import Quick
+import Nimble
+import Lipstick
+
+class UIColorModificatorsTest: QuickSpec {
+    
+    override func spec() {
+        describe("UIColor") {
+            var color: UIColor!
+            var white: UIColor!
+            beforeEach {
+                color = UIColor(red: 0.5, green: 0.1, blue: 0.1, alpha: 0.5)
+                white = UIColor(white: 0.5, alpha: 0.5)
+            }
+            describe("darker") {
+                it("works with rgb") {
+                    self.assert(UIColor(red: 0.25, green: 0.05, blue: 0.05, alpha: 0.5), color.darker(by: 50%))
+                }
+                it("works with white") {
+                    self.assert(UIColor(red: 0.25, green: 0.25, blue: 0.25, alpha: 0.5), white.darker(by: 50%))
+                }
+            }
+            describe("lighter") {
+                it("works with rgb") {
+                    self.assert(UIColor(red: 0.75, green: 0.15, blue: 0.15, alpha: 0.5), color.lighter(by: 50%))
+                }
+                it("works with white") {
+                    self.assert(UIColor(red: 0.75, green: 0.75, blue: 0.75, alpha: 0.5), white.lighter(by: 50%))
+                }
+            }
+            describe("saturated") {
+                it("works with rgb") {
+                    self.assert(UIColor(red: 0.5, green: 0, blue: 0, alpha: 0.5), color.saturated(by: 50%))
+                }
+                it("works with white") {
+                    self.assert(UIColor(red: 0.5, green: 0.5, blue: 0.5, alpha: 0.5), white.saturated(by: 50%))
+                }
+            }
+            describe("desaturated") {
+                it("works with rgb") {
+                    self.assert(UIColor(red: 0.5, green: 0.3, blue: 0.3, alpha: 0.5), color.desaturated(by: 50%))
+                }
+                it("works with white") {
+                    self.assert(UIColor(red: 0.5, green: 0.5, blue: 0.5, alpha: 0.5), white.desaturated(by: 50%))
+                }
+            }
+            describe("fadedIn") {
+                it("works with rgb") {
+                    self.assert(UIColor(red: 0.5, green: 0.1, blue: 0.1, alpha: 0.75), color.fadedIn(by: 50%))
+                }
+                it("works with white") {
+                    self.assert(UIColor(red: 0.5, green: 0.5, blue: 0.5, alpha: 0.75), white.fadedIn(by: 50%))
+                }
+            }
+            describe("fadedOut") {
+                it("works with rgb") {
+                    self.assert(UIColor(red: 0.5, green: 0.1, blue: 0.1, alpha: 0.25), color.fadedOut(by: 50%))
+                }
+                it("works with white") {
+                    self.assert(UIColor(red: 0.5, green: 0.5, blue: 0.5, alpha: 0.25), white.fadedOut(by: 50%))
```

Comment:
This doesn't seem right. "Faded out" should just decrease alpha, not the rest of the color. White (1, 1, 1, 1), faded out by 50% should be `(1, 1, 1, 0.5)`